tests/docker-build-vm.sh

```diff
@@ -23,11 +24,17 @@ MIRROR='https://deb.debian.org/debian'
 echo " ****************************************************************** "
 echo " * Running grml-debootstrap"
 
+if [ "$RASPI" = 'yes' ]; then
+  extra_buildopts='--rpifile --non-free'
+else
+  extra_buildopts='--vmfile'
+fi
+
 grml-debootstrap \
   --debug \
   --force \
-  --vmfile \
-  --vmsize 3G \
+  ${extra_buildopts} \
+  --imagesize 3G \
```

Comment:
Hm, is there a good reason why we couldn't continue using `--vmsize` and need to replace it with `--imagesize`? :thinking:

Reply:
We could keep using `--vmsize` for sure, but I wanted to switch to `--imagesize` since now that the argument can be used for more than just VM images, I marked `--vmsize` as deprecated. They mean the same thing, I figured using the newer syntax would be less confusing, especially since this build command might be building an RPi image, not strictly a VM image (though it will later boot that not-really-a-VM-image in a VM so I guess it doesn't really matter that much).

Reply:
Fixed the environment variable issue. Will leave this thread open so zeha can comment on the `--vmsize`/`--imagesize` stuff.